ThunderCloud/AppCollectionCell.swift

```diff
@@ -103,7 +104,7 @@ extension AppCollectionCell {
 				handler: { (action) in
 					
 					NotificationCenter.default.sendStatEventNotification(category: "Collect them all", action: "Open", label: nil, value: nil, object: self)
-					UIApplication.shared.open(launchURL, options: [:], completionHandler: nil)
+					UIApplication.shared.open(launchURL, options: convertToUIApplicationOpenExternalURLOptionsKeyDictionary([:]), completionHandler: nil)
```

Comment:
Take the options param out as it isn't used. Then scrap the convert method

Reply:
Eurghhh, stupid migration tool!